site/src/pages/WorkspacePage/WorkspaceNotifications/WorkspaceNotifications.tsx

```diff
@@ -254,3 +273,18 @@ const styles = {
 		gap: 12,
 	},
 } satisfies Record<string, Interpolation<Theme>>;
+
+const findTroubleshootingURL = (
+	workspaceBuild: WorkspaceBuild,
+): string | undefined => {
+	for (const resource of workspaceBuild.resources) {
+		if (resource.agents) {
+			for (const agent of resource.agents) {
+				if (agent.troubleshooting_url) {
+					return agent.troubleshooting_url;
```

Comment:
non-blocking: I can imagine a situation where a multi-agent workspace has a different troubleshooting URL for each agent. We might need some design changes to show multiple troubleshooting URLs though.

Reply:
I thought about this case! I concluded that having different Troubleshooting guides for various agents is an edge case.